content/departments/engineering/teams/core-services/managed-services/platform.md

```diff
@@ -22,3 +53,7 @@ This role is created org-level [in `gcp/org/customer-roles/msp.tf` in the infras
   - Resource: name of MSP project you are interested in
   - Role: `mspServiceEditor`
   - Duration: choose your own adventure!
+
+### Terraform Cloud
+
+Terraform Cloud workspaces for MSP [can be found using the `msp` workspace tag](https://app.terraform.io/app/sourcegraph/workspaces?tag=msp).
```

Comment:
are we expecting people to use `cli` mode often? right now no one but `owner` on TFC can apply these workspaces.

I have been hacking it by using `owner` role through Entitle.

Reply:
No, I don't anticipate the stuff here will be used often, it's just for development

I'm able to apply without entitle, and I use cli mode extensively 🤔

Reply:
That said these workspaces in VCS mode are mostly a reference